src/components/ItaliaTheme/Blocks/VideoGallery/Sidebar.jsx

```diff
@@ -88,6 +92,17 @@ const Sidebar = ({
               });
             }}
           />
+          <CheckboxWidget
```

Comment:
non so se avrei messo la checkbox per l'utente.. 
forse è meno macchinoso per il redattore inserire un url qualsiasi, e poi si arrangia il componente a mostrare l'embed corretto youtube/vimeo/altro. In questo modo sparirebbe il messaggio 'è un un url non valido..ecc'. Potenzialmente se mettono il check su 'allowExternal' poi possono mettere anche url non validi.. 
Ma parliamone